.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,29 +1,30 @@
 ---
 name: Bug report
 about: Create a report to help us improve
+labels: bug
 
 ---
 
-**Describe the bug**
+## Describe the bug
 A clear and concise description of what the bug is.
 
-**To Reproduce**
+## To Reproduce
 Steps to reproduce the behavior OR commands run:
 1. Go to '...'
 2. Click on '....'
 3. Enter value '...'
 4. See error
 
-**Expected behavior**
+## Expected behavior
```

Comment:
Can we change this to behaviour? My spellcheck is going nuts today. 😄

Reply:
Fair enough